packages/manager/apps/pci-volume-snapshot/src/pages/onboarding/Onboarding.page.spec.tsx

```diff
@@ -90,4 +90,21 @@ describe('OnBoardingPage', () => {
       'pci_projects_project_storages_snapshots_title',
     );
   });
+
+  it('should render guides', () => {
+    const { getByText } = render(<OnBoardingPage />, {
+      wrapper: createWrapper(),
+    });
+
+    expect(
+      getByText(
+        'pci_projects_project_storages_snapshots_onboarding_guides_transfer-volume-backup_title',
+      ),
+    ).toBeVisible();
+    expect(
+      getByText(
+        'pci_projects_project_storages_snapshots_onboarding_guides_transfer-volume-backup_description',
```

Comment:
also, this one is still available : `pci_projects_project_storages_snapshots_onboarding_guides_transfer` => is it not replaced by `pci_projects_project_storages_snapshots_onboarding_guides_${guide.id}_title & _description`?